### ISSUE TITLE
Work is impermanent / doesn't save

### ISSUE THREAD

**Issue by ktboundary65ma** [ChristopherChudzicki/math3d-react#291]
First: thank you for creating this beautiful & powerful visualization tool! However, it seems that whenever I move away from my work (e.g. click on the "Feature Demos" or "Neat Examples" tabs) my plot is lost and I can't recover it. What is the correct workflow for building and saving my own plots? In my opinion, a Desmos-style "saved graphs" interface would be ideal.

**Comment by ChristopherChudzicki** [comment]
I agree—a "saved graphs" tab would be great. This would require a user login system, which math3d doesn't have right now. (And would be a significant task that I don't have time for at the moment.)

In the meantime, you can click the "Share" button in upper-right-hand corner to get a shareable link.

The link is permanent but you do you have manage storing the link yourself =/

**Comment by o-b-o** [comment]
hey, would it be possible to share the database of graphs?
I'm sure it's huge, but there must be some gold in there. 
I don't reckon you need a user function to do something like that.

**Comment by ja1den** [comment]
> This would require a user login system, ...

Would it be less effort to save graphs to localStorage?
